### PR TITLE
PAPERDRIVE-1: Implement usable email template for reset password - Common-utils

### PR DESCRIPTION
**Pull request by deveshk1**
PAPERDRIVE-1: Implement usable email template for reset password - logo not visible

PAPERDRIVE-1: Implement usable email template for reset password - added paperdrive next to logo

image

### CODE REVIEW

File: middlewares/auth/index.ts [hunk 1]
```diff
@@ -343,7 +343,8 @@ export function createAuthMiddleware(
                 return
             }
             const secret = Utils.generateRandomID(20)
-            const host = req.get('host') || req.hostname;
+            // const host = req.get('host') || req.hostname;
+            const host = 'localhost:8081'
```
Comment (deveshk1) [inline middlewares/auth/index.ts]: Revert

File: middlewares/auth/mail/images/image-1.png
Comment (deveshk1) [inline middlewares/auth/mail/images/image-1.png:1]: is this required?

File: middlewares/auth/mail/images/image-2.png
Comment (deveshk1) [inline middlewares/auth/mail/images/image-2.png:1]: is this rquired?

File: middlewares/auth/mail/images/image-3.png
Comment (deveshk1) [inline middlewares/auth/mail/images/image-3.png:1]: is this rquired?

File: middlewares/auth/mail/images/image-6.png
Comment (deveshk1) [inline middlewares/auth/mail/images/image-6.png:1]: is this rquired?

File: middlewares/auth/mail/mailer.ts [hunk 2]
```diff
@@ -46,35 +46,40 @@ export class Mailer {
...
-        let htmlBody = this.getEmailTemplate();
+    async sendTextEmail(to: string, title: string, reset_link: string, username?: string,cc?: string) {
+        let htmlBody = this.getEmailTemplate(username);
 
         let mailOptions = {
             from: `'${this.config.senderName}' <${this.config.email}>`,
             to: to,
             subject: title,
-            html: htmlBody.replace("{{body}}", replaceAll(body, '\n', '<br>')),
+            html: htmlBody.replace("{{reset_link}}", replaceAll(reset_link, '\n', '<br>')),
```
Comment (deveshk1) [inline middlewares/auth/mail/mailer.ts]: ??

File: middlewares/auth/index.ts (hunk 1 above)
Comment (semibittechnologies) [reply middlewares/auth/index.ts]: done

**Review by shiveshnavin — APPROVED** [review]
LGTM

**Pull request merged**